zap/src/main/java/org/zaproxy/zap/model/Tech.java

```diff
@@ -287,4 +288,28 @@ public static Tech get(String name) {
         }
         return null;
     }
+
+    /**
+     * Gets the Techs that match the name or for which the parent matches the name, or an empty Set.
+     *
+     * @param name the name of the Tech
+     * @return the matching Techs
+     * @since 2.14.0
+     */
+    public static Set<Tech> getTechs(Tech t) {
```

Comment:
I'm wondering if this should be behaviour of `TechSet`, e.g. `includeAll` or `includeTree`.